compiles/ACG721Compile.js

```diff
@@ -0,0 +1,38 @@
+// to compile solicity source code and export it as an ABI (interface) for accessing
+const path = require('path');   //get cross platform
+const fs = require('fs');
+const solc = require('solc');
+
+// create a path to the solidity file
+const testTokenPath = path.resolve(__dirname, '..', 'contracts', 'acg721.sol');
+const testLibPath = path.resolve(__dirname, '..', 'contracts', 'SafeMath.sol');
+const testACG20Path = path.resolve(__dirname, '..', 'contracts', 'acg20.sol');
+const input = {
+    'acg721.sol': fs.readFileSync(testTokenPath, 'utf8'),
+    'acg20.sol': fs.readFileSync(testTokenPath, 'utf8'),
+    'SafeMath.sol': fs.readFileSync(testLibPath, 'utf8')
+};
+
+compilingResult = solc.compile({sources: input}, 1, (path) => {
+    // Solc doesn't support importing from other folders
+    // so resolve the missing files here
+    if (path == "helpers/SafeMath.sol") {

```

Comment:
If we move SafeMath.sol out of helpers folder, then we don't need to keep this operation here.

Reply:
And this script is used to compile all contracts but not ACG721 only, so I suggest changing its current filename.